### PR TITLE
Serialized saves

### PR DESCRIPTION
**Pull request by variousred**
working on specs

### CODE REVIEW

**Comment by variousred** [comment]
deployed to
https://mj-cms-1tf8b6q3-mitchell-prope.herokuapp.com/

File: lib/saves_manager.rb [hunk 1]
```diff
@@ -1,81 +1,84 @@
...
-  def backups_path
-    "#{prefix}/#{FOLDER_NAME}"
+  def upload
+    @uploaded_object = S3Uploader.new(@file_absolute_path, (backups_dir_name + @file_name), s3_bucket, write_options).upload
   end
 
-  def bucket_backups_full_path
-    "#{s3_bucket.name}/#{backups_path}"
+  def create_save
+    file_root = WebsiteSeederSerializer.new(location).to_yaml_file(DateTime.now.strftime("%Y%m%dT%H%M") + user_email)
```
Comment (thescubageek) [inline lib/saves_manager.rb]: This isn't going to work for backups! WebsiteSeederSerializer will create a generic "seeder" version of the site, NOT an actual full copy of all the widgets and their settings.....

Comment (thescubageek) [inline lib/saves_manager.rb]: I've updated this to work using the new deep_serialize option....

**Comment by variousred** [comment]
@thescubageek something with the deep merges is causing the seeded website to have a web_theme that does not have a garden_web_theme id, and failing a bunch of specs.
```LiquidParameters#theme_liquid``` is looking for
```      "theme_slug"                => liquid_web_theme.garden_web_theme.slug,
```
and not finding the garden web theme.
I'm not sure how those changes caused this to happen

File: lib/seeder/web_layout_seeder.rb [hunk 2]
```diff
@@ -24,4 +24,5 @@ def layout_params
       params_for(GardenWebLayout, @instructions, :garden_web_layout_id)
     end
   end
-end
\ No newline at end of file
+end
+
```
Comment (variousred) [inline lib/seeder/web_layout_seeder.rb]: @thescubageek this was deleted on the latest master, is it needed here?

Comment (thescubageek) [inline lib/seeder/web_layout_seeder.rb]: Web Layout seeder (and all stuff associated with Layout Garden) has been removed.

File: app/controllers/api/v1/saves_controller.rb [hunk 3]
```diff
@@ -1,23 +1,31 @@
...
+    location_id = params[:location_id]
+    location = Location.find(location_id)
+    render json: saves_manager(location).fetch_all, each_serializer: SaveSerializer
   end
 
   def restore
-    #safe_id — dont ask
-    saves_manager.restore(params[:safe_id])
+    Resque.enqueue(WebsiteRestoreJob, current_user.email, params[:location_id], params[:safe_id])
+    Resque.enqueue(GardenWidgetUpdaterJob)
```
Comment (thescubageek) [inline app/controllers/api/v1/saves_controller.rb]: Let's just enqueue a PostDeployUpdaterJob, which does both widgets and themes....

**Comment by variousred** [comment]
@thescubageek ready to go

**Comment by thescubageek** [comment]
@variousred Great work, this has been merged into 2.1.0 and will :ship: this evening: https://github.com/g5search/g5-content-management-system/pull/760

**Pull request merged**